src/resolvers-cassandra/Settings/queries.js

```diff
@@ -52,11 +52,29 @@ function sites(args, res) { // eslint-disable-line no-unused-vars
 function twitterAccounts(args, res) { // eslint-disable-line no-unused-vars
 }
 
+function cassandraRowToTrustedTwitterAccount(row) {
+  return {
+    RowKey: `${row.connector},${row.sourceid},${row.sourcetype}`,
```

Comment:
In the AzureTable world, this was the primary key of the database row, which IIRC was a UUID. In Cassandra, the key is a compound primary key made of of the `connector`, `sourceid`, and `sourcetype` columns:

```
CREATE TABLE fortis.trustedsources (
    connector text,
    sourceid text,
    sourcetype text,
    insertion_time timestamp,
    rank int,
    PRIMARY KEY (connector, sourceid, sourcetype)
...
```